spec/controllers/answers_controller_spec.rb

```diff
@@ -46,6 +46,34 @@
     end
   end
 
+  describe "PATCH#approve" do
+    before { sign_in login_user }
+
+    it "質問者であれば、回答を承認できること" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.